src/test/java/io/reactivex/schedulers/SchedulerTest.java

```diff
@@ -1,48 +1,48 @@
 /**
  * Copyright (c) 2016-present, RxJava Contributors.
- *
+ * <p>
```

Comment:
Please restore the header to the standard.

Reply:
Sorry, I don't have yours code style and have still the same problem with formating.

Reply:
I corrected it.